README.md

```diff
@@ -74,6 +74,8 @@ Web version is also available now. but only transmitter version.
 - [Looking For Job](https://github.com/euphony-io/looking-for-job) : Euphony Demo to send and receive resumes.
 - [Listen My Order](https://github.com/euphony-io/listen-my-order) : Euphony Demo for communicate restaurant menu information.
 
+- [Wave-In (for store)](https://github.com/euphony-io/check-in-listener.git) : Euphony Demo for store to check-in using a safe number.
```

Comment:
As @designe 's comment in https://github.com/euphony-io/euphony/pull/113, how about fixing repository name and description?